<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>800</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>PointingHandCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTextBrowser" name="TEXT">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>671</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="imgLabel">
   <property name="geometry">
    <rect>
     <x>49</x> 
     <y>0</y>
     <width>800</width>
     <height>700</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lineWidth">
    <number>6</number>
   </property>
   <property name="text">
    <string/>
   </property>
</widget>
<widget class="QLabel" name="processedImgLabel">
    <property name="geometry">
        <rect>
            <x>49</x> 
            <y>0</y>
            <width>1000</width>
            <height>1000</height>
        </rect>
    </property>
    <property name="frameShape">
        <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
        <enum>QFrame::Raised</enum>
    </property>
    <property name="lineWidth">
        <number>6</number>
    </property>
    <property name="text">
        <string/>
    </property>
</widget>
  <widget class="QFrame" name="frameRight">
   <property name="geometry">
    <rect>
     <x>37</x>
     <y>77</y>
     <width>900</width>
     <height>550</height>##############################################3
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="SHOW">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>670</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Show</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CAPTURE">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>710</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Capture</string>
   </property>
  </widget>
  <widget class="QPushButton" name="QUIT">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>660</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>240</red>
         <green>2</green>
         <blue>2</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>248</green>
         <blue>44</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="PlaceholderText">
       <brush brushstyle="SolidPattern">
        <color alpha="128">
         <red>255</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>240</red>
         <green>2</green>
         <blue>2</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>248</green>
         <blue>44</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="PlaceholderText">
       <brush brushstyle="SolidPattern">
        <color alpha="128">
         <red>255</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>240</red>
         <green>2</green>
         <blue>2</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>120</red>
         <green>120</green>
         <blue>120</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="PlaceholderText">
       <brush brushstyle="SolidPattern">
        <color alpha="128">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CONTINUE">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>660</y>
     <width>111</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Continue</string>
   </property>
  </widget>
  <widget class="QPushButton" name="UPLOAD">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>700</y>
     <width>111</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Upload</string>
   </property>
</widget>
 </widget>
 <resources/>
 <connections/>
</ui>
